src/harness/testcases/registration_testcase.py

```diff
@@ -108,6 +108,54 @@ def test_WINFF_FT_S_REG_6(self):
     self.assertEqual(response['response']['responseCode'], 0)
 
   @winnforum_testcase
+  def test_WINFF_FT_S_REG_9(self):
+    """ Array Re-registration of Single-step-registered CBSD (CBSD ID exists)
+
+    The response should be SUCCESS.
+    """
+
+    # Register the device
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+
+    # Inject FCC IDs
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_c['fccId']})
+
+    # Make sure measCapability contains no value for all array elements
+    device_a['measCapability'] = []
+    device_b['measCapability'] = []
+    device_c['measCapability'] = []
+
+    # Register two devices
+    devices = [device_a, device_b]
+    request = {'registrationRequest': devices}
```

Comment:
Since you only use `devices` once, you could just do `request = {'registrationRequest', [device_a, device_b]}` and save yourself the delete later. It's not really a big deal either way though.